### PR TITLE
Add feedback page

### PR DESCRIPTION
**Pull request by JaneEllison**
Add feedback page with usung react-hook-form.

### CODE REVIEW

File: src/components/RatingInput/RatingInput.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import styles from './RatingInput.module.scss';
+
+function RatingInput({ onChange }) {
+  return (
+    <div className={styles.starRatingGroup} onChange={onChange}>
```
Comment (anton-vlasik) [inline src/components/RatingInput/RatingInput.jsx]: Looks pretty hardcoded to me... what if I want to go further and use 10 stars system of raiting?

File: src/components/Line/Line.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import styles from './Line.module.scss';
+
+function Line({ type, theme }) {
+  return <span className={[styles.verticalLine, styles[type], styles[theme]].join(' ')}></span>;
+}
+
+export default Line;
```
Comment (anton-vlasik) [inline src/components/Line/Line.jsx:15]: Prop types  - where are they?

File: src/components/RatingInput/RatingInput.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      <label htmlFor="rating-3"></label>
+      <input type="radio" id="rating-2" name="rating" value="2" />
+      <label htmlFor="rating-2"></label>
+      <input type="radio" id="rating-1" name="rating" value="1" />
+      <label htmlFor="rating-1"></label>
+    </div>
+  );
+}
+
+export default RatingInput;
```
Comment (anton-vlasik) [inline src/components/RatingInput/RatingInput.jsx:29]: Prop types ?

File: src/components/RatingInput/RatingInput.jsx (hunk 1 above)
Comment (JaneEllison) [reply src/components/RatingInput/RatingInput.jsx]: I refactored it

File: src/components/Line/Line.jsx (hunk 2 above)
Comment (JaneEllison) [reply src/components/Line/Line.jsx:15]: thank you, added

File: src/components/RatingInput/RatingInput.jsx (hunk 3 above)
Comment (JaneEllison) [reply src/components/RatingInput/RatingInput.jsx:29]: thank you, added

File: src/components/RatingInput/RatingInput.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+import styles from './RatingInput.module.scss';
+import { NUMBER_OF_STARS } from '../../constants/form';
+import Star from '../Star/Star';
+
+function RatingInput({ onChange }) {
+  return (
+    <div className={styles.starRatingGroup} onChange={onChange}>
+      {[...Array(NUMBER_OF_STARS).keys()]
```
Comment (anton-vlasik) [inline src/components/RatingInput/RatingInput.jsx]: This looks like an overhead. `[...new Array(NUMBER_OF_STARS)].map((_, val) => ...)` should be enough

File: src/components/RatingInput/RatingInput.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+import styles from './RatingInput.module.scss';
+import { NUMBER_OF_STARS } from '../../constants/form';
+import Star from '../Star/Star';
+
+function RatingInput({ onChange }) {
+  return (
+    <div className={styles.starRatingGroup} onChange={onChange}>
```
Comment (anton-vlasik) [inline src/components/RatingInput/RatingInput.jsx]: onChange? uummmm... please try to explain yourself :)

File: src/components/Star/Star.jsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+function Star({ value }) {
+  const starId = ['rating', value].join('-');
+
+  return (
+    <>
+      <input type="radio" id={starId} name="rating" value={value} />
```
Comment (anton-vlasik) [inline src/components/Star/Star.jsx]: `<input type="radio" id={starId} name="rating" value={value}  />`

File: src/components/RatingInput/RatingInput.jsx (hunk 4 above)
Comment (JaneEllison) [reply src/components/RatingInput/RatingInput.jsx]: Fixed.

File: src/components/RatingInput/RatingInput.jsx (hunk 5 above)
Comment (JaneEllison) [reply src/components/RatingInput/RatingInput.jsx]: I have refactored all RatingInput component and fixed it.

File: src/views/Feedback/FeedbackForm/FeedbackForm.jsx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    );
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} className={styles.feedbackForm}>
+      <Controller
+        control={control}
+        name="rating"
+        rules={{ required: true }}
+        render={({ field: { onChange, value } }) => (
```
Comment (anton-vlasik) [inline src/views/Feedback/FeedbackForm/FeedbackForm.jsx:33]: Much better now. And clearer!

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**